server/betterbin.js

```diff
@@ -392,6 +396,51 @@ Meteor.startup(function(){
 			}
 		}
 	});
-	console.log("Restivus has been started");
+	console.log("Restivus AoE2 api has been started");
+
+	//Restivus API for AoE2
+	var Api = new Restivus({
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Variable 'Api' is already declared in line 244.](https://www.codacy.com/app/igougi-ui/Mrb-in/file/2345231608/issues/source?bid=2882324&fileBranchId=3214315#l402)